static/news.en.html

```diff
@@ -1,12 +1,29 @@
 <h1 id="helpPage_newsTitle">News</h1>
 
+<h2 class="title">A new game mode, choice of complexity and more</h2>
+<span class="date">
+    __.08.2020
+</span><span class="version">
+    v1.2
+</span><span class="new">
+    * New
+</span>
+<p>
+The game has become more comfortable and interesting!
+
+<li>A new game mode with words from players: every participant writes words from which the set of words forms</li>
+<li>The Russian dictionaries with different complexity appeared</li>
+<li>If you have your own dictionary, you can upload it and play with words from it</li>
```

Comment:
Согласование с русской версией:
```suggestion
<li>Now you can upload your own dictionary and play with words from it</li>
```

Reply:
У меня вопрос, как лучше: "You can now..." или "Now you can..."? В прошлых новостях был первый вариант!

Reply:
Не уверен, но кажется, и так, и так.